.gitignore

```diff
@@ -61,4 +61,4 @@ tikv-client/proto
 
 # ignore tikv-client bazel build
 tikv-client/bazel-*
-
+data
```

Comment:
Is this line necessary?

Reply:
Yes. data will be created by docker. By ignoring this directory, we can avoid `git add` this directory.